2014-06-12  Jakub Jelinek  <jakub@redhat.com>

	PR plugins/45078
	* config.gcc (arm*-*-linux-*): Include vxworks-dummy.h in tm_file.

--- gcc/config.gcc.jj	2014-04-24 23:18:09.552816344 +0200
+++ gcc/config.gcc	2014-06-12 21:25:04.406828996 +0200
@@ -1021,7 +1021,7 @@ arm*-*-linux-*)			# ARM GNU/Linux with E
 	    ;;
 	esac
 	tmake_file="${tmake_file} arm/t-arm arm/t-arm-elf arm/t-bpabi arm/t-linux-eabi"
-	tm_file="$tm_file arm/bpabi.h arm/linux-eabi.h arm/aout.h arm/arm.h"
+	tm_file="$tm_file arm/bpabi.h arm/linux-eabi.h arm/aout.h vxworks-dummy.h arm/arm.h"
 	# Define multilib configuration for arm-linux-androideabi.
 	case ${target} in
 	*-androideabi)
